src/lib/button/button.scss

```diff
@@ -79,6 +80,10 @@
   border-radius: inherit;
   pointer-events: none;
   opacity: 0;
+
+  @include md-high-contrast {
+    background-color: rgba(white, 0.5);
```

Comment:
It probably would, but I only went for the dark modes in this PR since supporting all modes would add more bloat.